build.gradle

```diff
@@ -15,8 +15,8 @@ apply from: 'console.gradle'
 
 //apply from: file('setupTestsReports.gradle')
 
-sourceCompatibility = 14
-targetCompatibility = 14
+sourceCompatibility = 15
```

Comment:
Shouldn't we use '17'

Reply:
Good point. We probably should. I think in this case, it was so I could use Text Blocks.